### ISSUE TITLE
Wanted scenario - download a matrix file per one sample

### ISSUE THREAD

**Issue by gabsie** [HumanCellAtlas/dcp-feedback#6]
P4 wanted to just download a matrix file from one sample. (This was her task, and she only wanted to select one for a specific donor.)
She couldn't easily do the filtering on the interface to be able to do that task.
How she did it - from her manifest she downloaded a random matrix file for another random specimen, not the one she wanted (from the UI, let's say from the female donor)
Problem: we should be able to support this scenario. Participant wanted to explore both the Files and Specimen tabs to see if she can do it, using the facets as well, but she couldn't.
Solution - think how to support this scenario.

**Comment by NoopDog** [comment]
Yes, currently its not possible to restrict the selected file set by sample id. (you cant select a sample using the browser)

Once selecting a sample is in place this flow will be possible.
